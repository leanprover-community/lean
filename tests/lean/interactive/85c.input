{"seq_num":1,"command":"sync","file_name":"a.lean","content":".\n"}
{"seq_num":2,"command":"info","file_name":"a.lean","line":2,"column":0}
