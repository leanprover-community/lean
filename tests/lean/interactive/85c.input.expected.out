{"message":"file invalidated","response":"ok","seq_num":1}
{"response":"ok","seq_num":2}
